packages/list/mwc-list-base.ts

```diff
@@ -113,6 +131,29 @@ export abstract class ListBase extends BaseElement implements Layoutable {
   })
   noninteractive = false;
 
+  debouncedLayout: (updateItems?: boolean) => void | undefined;
```

Comment:
`protected debouncedLayout`

Reply:
this cannot be protected as it is used outside the class (in the items `disconnectedCallback`